geoana/kernels/tranverse_electric_reflections.py

```diff
@@ -0,0 +1,137 @@
+import numpy as np
+from scipy.constants import mu_0
+
+def _rTE_forward(frequencies, lamb, sigma, chi, thicknesses):
+    """Compute reflection coefficients for Transverse Electric (TE) mode.
+
+    The first layer is considered to be the top most layer. The last
+    layer is considered to have infinite thickness.
+
+    Parameters
+    ----------
+    frequencies : float, numpy.ndarray
+        Frequency (Hz); shape = (n_frequency, ).
+    lamb : float, numpy.ndarray
+        Spatial wavenumber (1/m), shape = (n_filter, ).
+    sigma: complex, numpy.ndarray
+        Conductivity (S/m), shape = (n_layer, n_frequency).
+    chi: complex, numpy.ndarray
```

Comment:
do we want to work with `chi` or `mu` here? I would be tempted to stick with `mu` throughout for electromagnetic problems